### ISSUE TITLE
Deprecated ViewProps

### ISSUE THREAD

**Issue by okb1100** [callstack/react-native-slider#431]
Hi! 👋 
      
Firstly, thanks for your work on this project! 🙂

Today I used [patch-package](https://github.com/ds300/patch-package) to patch `@react-native-community/slider@4.3.1` for the project I'm working on.

Please see https://github.com/facebook/react-native/issues/34688

Here is the diff that solved my problem:

```diff
diff --git a/node_modules/@react-native-community/slider/src/RNCSliderNativeComponent.js b/node_modules/@react-native-community/slider/src/RNCSliderNativeComponent.js
index 2075818..af8b817 100644
--- a/node_modules/@react-native-community/slider/src/RNCSliderNativeComponent.js
+++ b/node_modules/@react-native-community/slider/src/RNCSliderNativeComponent.js
@@ -13,7 +13,8 @@
 import type {HostComponent} from 'react-native';
 import type {ColorValue} from 'react-native/Libraries/StyleSheet/StyleSheetTypes';
 import type {ImageSource} from 'react-native/Libraries/Image/ImageSource';
-import type {ViewProps} from 'react-native/Libraries/Components/View/ViewPropTypes';
+// import type {ViewProps} from 'react-native/Libraries/Components/View/ViewPropTypes';
+import {ViewPropTypes as ViewProps} from 'deprecated-react-native-prop-types'
 import codegenNativeComponent from 'react-native/Libraries/Utilities/codegenNativeComponent';
 import type {
   Float,
diff --git a/node_modules/@react-native-community/slider/src/Slider.js b/node_modules/@react-native-community/slider/src/Slider.js
index 9d7a721..f30b1e1 100644
--- a/node_modules/@react-native-community/slider/src/Slider.js
+++ b/node_modules/@react-native-community/slider/src/Slider.js
@@ -24,7 +24,8 @@ import type {NativeComponent} from 'react-native/Libraries/Renderer/shims/ReactN
 import type {ImageSource} from 'react-native/Libraries/Image/ImageSource';
 import type {ViewStyleProp} from 'react-native/Libraries/StyleSheet/StyleSheet';
 import type {ColorValue} from 'react-native/Libraries/StyleSheet/StyleSheetTypes';
-import type {ViewProps} from 'react-native/Libraries/Components/View/ViewPropTypes';
+// import type {ViewProps} from 'react-native/Libraries/Components/View/ViewPropTypes';
+import {ViewPropTypes as ViewProps} from 'deprecated-react-native-prop-types'
 import type {SyntheticEvent} from 'react-native/Libraries/Types/CoreEventTypes';
 
 type Event = SyntheticEvent<
```

<em>This issue body was [partially generated by patch-package](https://github.com/ds300/patch-package/issues/296).</em>

**Comment by stianjensen** [comment]
Looks like this was solved in https://github.com/callstack/react-native-slider/pull/424, so we just need a new release with that :+1:

**Comment by BartoszKlonowski** [comment]
Fixed by #424
